### ISSUE TITLE
Blokada is on/active but does not block any adds (similar to #137 but other version)

### ISSUE THREAD

**Issue by Rek-Lama** [blokadaorg/blokada#764]
### Actual behaviour
Blokada is on/active but does not block any adds (similar to #137 but other version)

### Expected behaviour
Blokada is on, hostlists Energized Blu + Xiaomi-Blocker Jerryn70 are on. DNS is deactivated (but it also does not work too when it is acitvated with Quad9, Cloudflare, ...). The counter of blocked adds is still zero, no adds are blocked. It seems, that the hostlists are still ignorred and blokada is working "in transit". The log shows, that everything was allowed.

Device is with original Xiaomi Global ROM (bought directly by Xiaomy Germany). Bloadware is (partially) uninstalled/deactivated with Xiaomi ADB/Fastboot Tools (based on recommendations here: https://technastic.com/xiaomi-bloatware-list-miui/)

Device is *not* rooted.

On the same device with MIUI 11 Blokada worked fine. Since last Blokada-update it stopped to block adds (counter was still on ~370.000). Uninstall / reboot / re-install blokada does not help. Upgrade to MIUI 12 via OTA (Upgrade offered officially by Xiaomi). 
However, it seems that Blokada "remembers" my profile, that means after unistall, reboot device, re-install, blokada uses the same settings for hostlist and DNS, even if I do not have an account. How to clean uninstall blokada?

"downloads/Blokada/requests.csv" is created but it is empty.

### Steps to reproduce  
1. Start blokada on the described device
2. Use different apps (like wetter.com, ntv, ...)
3. Browse with different browsers (firefox, iron, ...) on different web-pages (like web.de, focus-online,.de, ...)
4. Result: all adds are still available, blokada counter still shows 0 blocked adds

### Configuration
1. Blokada version: 
blokada/4.7.3 (android-29 full official arm64-v8a Xiaomi-davinci touch api compatible) - German translation
2. Device model: 
Xiaomi 9T (Redmi K20), Global Version
3. Android version / ROM: 
Android 10 QKQ1.190825.002 / Security Update Level 2020-07-01
ROM MIUI Global 12.0.2 Stabil  12.0.2.0(QFJEUXM) - German translation

How to bring Blokada to work again?

### Questions
1. Do you know about recent updates (for example system update, security patch update)? 
Yes - Blokada and MIUI updated/upgraded (however, Blokada stopped before MIUI upgrade)

2.  How often does this problem happen to you?
Ongoing...

**Comment by peterroth** [comment]
Hi,

Can you check if you enable _Alternative DNS answer_ in Blokada's menu - DNS the ads are being blocked?
The system handles and cleans up the remaining files if you remove an application. Thus, if you remove Blokada, the system should remove everything that the app generated (except if you copy files from the original location to an other directory).

**Comment by Rek-Lama** [comment]
Hi peterroth,

thanks for your suggestion. I have tried all possible options:
1. DNS is active (Quad9), Alternative DNS answer is ON
2. DNS is active (Quad9), Alternative DNS answer is OFF
3. DNS is NOT active, Alternative DNS answer is ON
4. DNS is NOT active, Alternative DNS answer is OFF
No change in the behavior.

I did not copied/moved files from the default location of the installation.

Any other ideas?

Thanks
